src/Projector/ReadModelProxy.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Not entirely sure if this file and the next should be in here.

I would leave it in here, because it is a nice abstraction. This would immediately gives us a command to run/reset the projections.

Reply:
What does this class and does it needs to be final? In other words, can you override it?

Reply:
These two classes proxies the event streams to the projectors. (I think 😄 ).
They don't have to be final, so I have removed the final keyword.